hpat/tests/test_series.py

```diff
@@ -1033,8 +1032,7 @@ def test_impl(S, axis):
         hpat_func = hpat.jit(test_impl)
 
         S = pd.Series([1.0, 2.0, np.nan, 1.0, np.inf])
-        axis_values = [0, 'index']
-        for value in axis_values:
+        for value in [0, 'index']:
```

Comment:
I would name the loop variable as `axis`.

Reply:
Yes, that's changed too.

Reply:
@shssf @Hardcode84 Can you please approve and merge if you have no more comments?